source/includes/_assets.md

```diff
@@ -13,6 +13,7 @@ both years.
   - construction year
   - building certifications
   - asset ownership
+  - asset size
```

Comment:
`asset_size` is part of the annual data, not the building characteristics, although the UI shows it as such.

```
    "annual_data": [
        {
            "year": 2020,
            "asset_name": "The White House",
            "whole_building": true,
            "asset_size": 500
```